### PR TITLE
Add workshop guidelines

### PR DESCRIPTION
**Pull request by ad510**
Seriously, no one submitted a PR for this even though I basically wrote up the whole thing in #374 more than a week ago?

The reason I didn't submit a PR at the time was because the Hack Club repo has multiple guideline-like documents already, but I thought they gave redundant advice and had debatable value for people who want to contribute workshops, so I thought that a PR about workshop guidelines should clean those up too. But let's get my writeup in the repo first.

If you want to make changes before merging, go ahead and submit a new PR instead of asking me to make changes. I've been too busy with schoolwork to check on Hack Club regularly (though you guys are all awesome!)

### CODE REVIEW

**Comment by paked** [comment]
I'm going to assign @elishacoad as it looks like he was interested in working to merge this.

**Comment by ad510** [comment]
All @elishacoad had to do in #374 was make a pull request with the writeup I already wrote, but he didn't do it for more than a week so I made this pull request. Can you take care of merging this @paked?

**Comment by ad510** [comment]
@elishacoad I added Cmiel to the dictionary in https://github.com/hackclub/markcop/issues/81 so the tests should pass now if you re-run them. I believe the only things left to do are to read over the files changed to make sure they're good, manually rerun the tests, then if they pass you can press the "Merge pull request" button at https://github.com/hackclub/hackclub/pull/747.

@paked Can you tell @elishacoad how to rerun the tests? I saw @zachlatta do that on previous PRs but I don't know how he did it.

**Comment by paked** [comment]
Only @zachlatta has those powers, but when you push something they get ran
anyway. I generally trigger myself by force pushing :')

I'll reach out to @elishacoad in the Slack so we can get this moving!

**Comment by ad510** [comment]
Sorry @elishacoad about my previous comment (now deleted). I wrote that comment at something like 2 am because I've been completely occupied by schoolwork during the daytime, but that doesn't excuse me from conveying the wrong meaning. If it took you a long time to get to #374 because you are busy too, then I fully understand because that's the kind of situation I'm in too.

**Comment by MaxWofford** [comment]
@paked People with access to the circleci group can rebuild it. Not sure, but I think you can add yourself if you are part of the hackclub organization on GitHub.
screenshot from 2016-03-01 16 26 07

**Comment by ad510** [comment]
I ran `git commit --amend` without changing anything then did force pushed, so the tests pass now. Looking forward to getting this merged!

**Comment by elishacoad** [comment]
Hey, sorry guys for my inactivity, I've been real busy with school and hosting a hackathon. So what was it exactly that is needed to be done, I'm not too familiar with how Github works. I'm glad to help out however necessary. (I also have a rough outline for the seggusted framework of a workshop)

**Comment by elishacoad** [comment]
If you like, feel free to use this general outline to model your workshop:

1. Title - Opening
- State workshop's end result and estimated time
- Include final example
- Describe what will be learned on the way
- Table of contents

2. (Roman Numerals) Body Title - Split into parts
- Summary Statement
(Arabic Numerals)
1. Step
  •additional explanation and side-notes
-ending step- Celebrate! (For mile-stone Parts)
- say what they've accomplished

3. Closing
- list of expansion ideas
- any other notes

*modeled after Portfolio

File: workshops/GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+The answer to that is that we're not just trying to teach people how to code.
+We're trying to teach people how to become self-learning hackers. It is common
+for people to finish an online learn-to-code curriculum (or get a full-blown
+computer science college degree, for that matter), and still not understand why
+hacking together a project is so *fun*! We want to encourage the kind of
+mentality where you just randomly decide to code up a project over the weekend
+(for fun!) and brag about it to friends.
+
+For example, when you're making your own project over the weekend, you don't
+have a workshop page in front of you telling you how to solve your problem step
```
Comment (elishacoad) [inline workshops/GUIDELINES.md]: At the end it should be step-by-step.

**Comment by elishacoad** [comment]
All looks good! I not able to merge because  I don't have the proper permissions – can a member of the staff merge this?

**Comment by zachlatta** [comment]
Just assigned @remixz, he'll be able to take care of this :-).

**Comment by remixz** [comment]
Looks awesome! Great work everyone. :clap:

**Comment by paked** [comment]
:+1: HELL YEAH GUYS!

**Pull request merged**